projects/swimlane/ngx-ui/src/lib/components/drawer/drawer.component.ts

```diff
@@ -81,7 +80,7 @@ export class DrawerComponent implements OnInit, OnDestroy {
   transform: string;
   widthSize: string | number;
   heightSize: string | number;
-  position: 'fixed' | 'initial' = 'fixed';
+  position: 'fixed' | 'absolute' = 'absolute';
```

Comment:
can we change this to an enum instead of using magic strings?

Reply:
enumed it up